fulfillment-service/it/it_identity_providers_login_test.go

```
@@ -0,0 +1,256 @@
+				Metadata: privatev1.Metadata_builder{
+					Name:   idpName,
+					Tenant: tenantName,
+				}.Build(),
+				Spec: privatev1.IdentityProviderSpec_builder{
+					Title:   "Mock OIDC Test Provider",
+					Enabled: true,
+					Oidc: privatev1.OidcConfig_builder{
+						// authorizationUrl must be reachable from the test runner (host).
+						// mockoidc binds to 0.0.0.0 so 127.0.0.1:<port> works on the host.
+						AuthorizationUrl: mockOIDC.LocalAuthURL(),
+					// tokenUrl and jwksUrl must be reachable from Keycloak inside Kind.
```

golangci-lint (2.13.2)

[error] File is not properly formatted

(gofmt)